account_payment_order/views/ir_attachment.xml

```diff
@@ -12,12 +12,7 @@
                         <field name="name" />
                     </h1>
                     <group name="main">
-                        <field
```

Comment:
Can I ask if this alternative attachment view is in need of a value for the field `priority` so that it does not compete with the main attachment form view?

Reply:
it's possible but this FIX have 3 fix's and I don't want to add more. If the view haver any problem open other fix please.

Reply:
Yes, I was wondering about adding this second fix but given that you are modifying this view you could either split off all view fixes into a separate PR or add the priority + view selection in this PR.

Reply:
Si me dice que debo cambiar, puedo agregar los cambios en esta solución 👍

Reply:
Thanks, but you are right about combining too many different fixes in the same PR. I've proposed this priority thing here: https://github.com/OCA/bank-payment/pull/712/files